.gitignore

```diff
@@ -1,6 +1,7 @@
 # Generated by Cargo
 # will have compiled files and executables
 **/target/
+**/bin/
```

Comment:
Why does adding `bin` to `.gitignore` cover us from not committing the `result` folder?

Reply:
because `/bin` is the only directory and contents of `result`, so by ignoring `**/bin` we both ignore `result` and any other `bin` that may be outputted somewhere else / by other tool.